### ISSUE TITLE
Customize pull request templates?

### ISSUE THREAD

**Issue by climbfuji** [JCSDA/spack-stack#582]
**Is your feature request related to a problem? Please describe.**
spack-stack has its own issue templates, but no pull request templates. We are getting the JCSDA default pull request template now that the repository moved.

**Describe the solution you'd like**
We should check if the default template from JCSDA works for us or not, and if not create our own template.

**Comment by climbfuji** [comment]
@AlexanderRichert-NOAA @ulmononian Do you have any opinion on this spack-stack question?

**Comment by ulmononian** [comment]
All of this is very JCSDA/JEDI-specific:

```

If changes in this PR will affect other repositories, please add a "waiting for other repos" label, and list the repositories that will be affected to the best of your knowledge (example below).
Requires changes in the following repositories:
- [ ] saber
- [ ] ioda
- [ ] ufo
- [ ] ...

If changes in this PR require updating test data on AWS please add an "update test data" label and list the test data that needs to be updated to the best of your knowledge (example below).
Requires updating AWS test data for the following repositories:
- [ ] saber
- [ ] ioda
- [ ] ...

Note: to automatically run saber, ioda and ufo tests with this PR, push a commit containing "trigger pipeline", e.g.:
git commit --allow-empty -m 'trigger pipeline'
```

also, isn' it more likely to have dependencies on spack prs rather than other jcsda repos in the `Dependencies` section? currently it looks like:

```
Waiting on the following PRs:
- waiting on JCSDA/eckit/pull/<pr_number>
- waiting on JCSDA/atlas/pull/<pr_number>
```

but it might be a better default to be:

```
Waiting on the following PRs:
- waiting on JCSDA/spack/pull/<pr_number>
```
i'm not attached to these suggestions, but since you asked...

**Comment by climbfuji** [comment]
I agree - I created those issues to gather input from the spack-stack developers (and maybe find a volunteer to create our own spack/spack-stack PR templates ;-) ).

**Comment by AlexanderRichert-NOAA** [comment]
I'd be happy to start a draft of a PR template to give us something concrete to discuss, and maybe plan on going over it in the next release meeting.

**Comment by climbfuji** [comment]
That sounds great, thanks very much @AlexanderRichert-NOAA - if that all happens while I am on leave, please include @srherbener. Thanks again.

**Comment by shlyaeva** [comment]
@ulmononian note that the template changed significantly yesterday, and no longer has those lines.

**UPDATE**: the template changed in JCSDA-internal, but not in JCSDA.

**Comment by srherbener** [comment]
I did a PR on Tuesday (5/30/23) and it seemed that the JEDI specific aspects have been removed (as if JCSDA got sync'd up with JSCDA-internal). The PR template I got on Tuesday seems good to me.

Where is the PR template located? Just want to double check it got updated. Thanks!

**Comment by shlyaeva** [comment]
The PR template is located here: https://github.com/JCSDA/.github; indeed we updated the template to be the same as in JCSDA-internal (https://github.com/JCSDA/.github/pull/1)

**Comment by srherbener** [comment]
Thanks @shlyaeva, I can see it now. I vote to start with this one and evolve it as needed.

**Comment by AlexanderRichert-NOAA** [comment]
@srherbener I started drafting a new one here-- https://github.com/AlexanderRichert-NOAA/spack-stack/tree/pr_template
I've put this on the agenda for today's spack-stack meeting.
